README.md

```diff
@@ -1 +1,97 @@
-# presto-alibaba-maxcompute-connector
+# MaxCompute Connector
+
+The MaxCompute connector enables direct querying and processing of data stored in the MaxCompute data warehouse. This connector is useful for implementing data
+integration and analysis between MaxCompute and other systems, such as Hive.
+
+## MaxCompute Connector Setup
+
+### Prerequisites
+
+Before using the MaxCompute connector, ensure the following conditions are met:
+
+- Access permissions: You must have valid access rights to a MaxCompute project and the necessary authentication credentials for that project.
+- Network setup: For stable data transmission, the use of Aliyun VPC network is recommended. See
+  the [MaxCompute Network Configuration Guide](https://help.aliyun.com/zh/maxcompute/user-guide/network-connection-process) for more information.
+- Resource group configuration: Possession of an exclusive Tunnel resource group is required. Note: MaxCompute currently allows access to shared resource groups, but this could
+  change at any time. See [Exclusive resource group](https://help.aliyun.com/zh/maxcompute/user-guide/purchase-and-use-exclusive-resource-groups-for-dts).
+- (Optional) Schema activation: Enable Schema mode to use different directories in Presto. See
+  [How to Enable Schema Model in MaxCompute](https://help.aliyun.com/zh/maxcompute/user-guide/schema-related-operations) for more information.
+
+### Obtain the Connector JAR
+
+Obtain the JAR file by building the repository. Use the Maven command `mvn clean package` to build the project and generate the
+JAR. Once built, place the JAR file into the `plugin/maxcompute` directory within your Presto installation.
+
+### Configuration
+
+Before configuring the MaxCompute connector, have ready all the necessary information associated with your MaxCompute project, including project name, access
+ID, access key, and resource group name.
+
+1. Create a configuration file (for example, `maxcompute.properties`) in Presto's `etc/catalog` directory:
+
+```properties
```

Comment:
Suggest indenting the code block so that it displays within item 1. As it is, the code block is flush with the left margin. You can check this with "View file" in GitHub, which is in the menu displayed when "..." in the upper right of this file is selected.